### PR TITLE
reporting functions for pagerank

### PR DESCRIPTION
**Pull request by whasenpl**

### CODE REVIEW

File: src/graph_compute/numa_scheduling.h [hunk 1]
```diff
@@ -183,14 +183,16 @@ static inline void createChunkData(vertex_t * const nodes,
   scheddata->numChunksPerWorker =
     (scheddata->cntChunks + NUMA_WORKERS - 1) / NUMA_WORKERS;
 
-  for (vid_t i = 0; i < scheddata->cntChunks; ++i) {
+  cilk_for (vid_t i = 0; i < scheddata->cntChunks; ++i) {
```
Comment (obi1kenobi) [inline src/graph_compute/numa_scheduling.h:186]: Are we using Cilk in NUMA? I thought we were avoiding it as a precaution against weird interactions with pthreads.

**Comment by obi1kenobi** [comment]
looks good other than the one comment I made -- if using cilk in NUMA is intentional, feel free to merge.

File: src/graph_compute/numa_scheduling.h (hunk 1 above)
Comment (obi1kenobi) [inline src/graph_compute/numa_scheduling.h:186]: @whasenpl thoughts on this? I'd like to move this to master soon.

Comment (whasenpl) [inline src/graph_compute/numa_scheduling.h:186]: Sorry, I thought we resolved this.  I learned from Charles and Bradley that there is no problem using Cilk and pthreads in mutually exclusive portions of the code.  Technically, you're supposed to be able to use pthreads and Cilk concurrently, but that's a little dodgier.

**Pull request merged**